integrations_core_downloader/README.md

```diff
@@ -0,0 +1,3 @@
+# integrations-core-downloader
+
+A Datadog-specific secure downloader for integrations-core Python packages
```

Comment:
Is this only for `core` (even long term)?

Reply:
`core` only for now, but i will make it clear in doc